.zuul.d/jobs.yaml

```diff
@@ -98,6 +98,15 @@
       authentication enabled.
     run: tests/with_client_cert.yaml
 
+- job:
+    name: ara-role-api-podman
+    parent: ara-role-integration-base
+    nodeset: ara-multinode
+    description: |
+      Desploys the ARA API server on Fedora 36 as well as CentOS Stream 8/9
+      in a Podman container and tests it using the default sqlite database backend.
+    run: tests/with_podman.yaml
```

Comment:
Cool! We should bump that to Fedora 38 like I did recently for [ara](https://github.com/ansible-community/ara/commit/96db662715b1d668617cab2626703347127f89bf) but I can take care of that in another PR.

Reply:
Let's split that, first let's make this work ^^

Reply:
I like that there is already a job for testing podman and that it is even passing, nice work :)